### PR TITLE
disable flaky tests

### PR DESCRIPTION
**Pull request by nsinkov**
## Changes proposed in this PR

disable flaky tests

## Why are we making these changes?

these tests are flaky

### CODE REVIEW

File: waiter/integration/waiter/request_timeout_test.clj [hunk 1]
```diff
@@ -137,7 +137,8 @@
       (make-request-and-assert-timeout waiter-url request-headers start-time-ms timeout-period-sec)
       (delete-service waiter-url request-headers))))
 
-(deftest ^:parallel ^:integration-slow test-request-queue-timeout-max-instances-limit
+; Marked explicit because it's flaky
```
Comment (dposada) [inline waiter/integration/waiter/request_timeout_test.clj]: When possible, we like to add some context on the flakiness

Comment (nsinkov) [reply waiter/integration/waiter/request_timeout_test.clj]: Hard to exfiltrate from Jenkins :p

Comment (dposada) [reply waiter/integration/waiter/request_timeout_test.clj]: Maybe just a brief explanation of the nature of the flakiness, if we know it?

**Comment by nsinkov** [comment]
@shamsimam 

This test was flaky when I was deploying to production and is still flaky on a new branch I'm working on.

The plan is to add flaky test detection on the master branch and escalate new flaky tests to the support person.

OK to merge?

**Pull request merged**